lib/pdn/pdn.c

```diff
@@ -659,6 +659,51 @@ int pdn_dynamic_params_get(uint8_t cid, struct in_addr *dns4_pri,
 	return 0;
 }
 
+int pdn_dynamic_params_get_v6(uint8_t cid, struct in6_addr *dns6_pri,
+			   struct in6_addr *dns6_sec, unsigned int *ipv6_mtu)
+{
+	int matched;
+	const char *fmt;
+	unsigned int mtu;
+	char dns6_pri_str[INET6_ADDRSTRLEN];
+	char dns6_sec_str[INET6_ADDRSTRLEN];
+	char at_cmd[sizeof("AT+CGCONTRDP=10")];
+
+	if (snprintf(at_cmd, sizeof(at_cmd), "AT+CGCONTRDP=%u", cid) >= sizeof(at_cmd)) {
+		return -E2BIG;
+	}
+	/* "+CGCONTRDP: 0,,"ims","","","0000:0000:0000:0000:0000:0000:0000:0000","0000:0000:0000:0000:0000:0000:0000:0000",,,,,1500" */
+	fmt = "+CGCONTRDP: %*u,,\"%*[^\"]\",\"\",\"\",\"%45[0-9A-Fa-f:]\",\"%45[0-9A-Fa-f:]\",,,,,%u";
```

Comment:
```suggestion
	fmt = "+CGCONTRDP: %*u,,\"%*[^\"]\",\"\",\"\","
		"\"%45[0-9A-Fa-f:]\",\"%45[0-9A-Fa-f:]\",,,,,%u";
```
You can split the string to stay within the maximum line length.

Reply:
Ok, thanks.